src/components/ui/Animation.js

```diff
@@ -275,12 +266,14 @@ CIQ.Animation = function (stx, animationParameters, easeMachine) {
             if (this.chart.yAxis.left > x &&
                 this.chart.yAxis.top <= y &&
                 this.chart.yAxis.bottom >= y) {
-                if (flashingColorIndex >= flashingColors.length) { flashingColorIndex = 0; }
-                context.beginPath();
-                context.moveTo(x, y);
-                context.arc(x, y, 2 + flashingColorIndex * 1.07, 0, Math.PI * 2, false);
-                context.fillStyle = flashingColors[flashingColorIndex];
-                context.fill();
+                if (!currentSpotElement) {
```

Comment:
if spot not exists, then add it to DOM, if exists then just update the `Top, Left` attributes.